From: Ichthyostega <prg@ichthyostega.de>
Date: Mon, 16 Jan 2017 03:22:33 +0100
Subject: =?utf-8?q?fix_include_f=C3=BCr_QWebView_to_work_with_older_QT-5_v?=
 =?utf-8?q?ersions?=

Prior to QT 5.6, you'd still use the old QWevView header include,
but it is already moved down into the QtWebKitWidgets package
---
 CMakeLists.txt            | 1 +
 src/widgets/helpbrowser.h | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ab9308d..ed2e50f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -85,6 +85,7 @@ FIND_PACKAGE(Qt5Widgets REQUIRED)
 #FIND_PACKAGE(Qt5LinguistTools REQUIRED)
 FIND_PACKAGE(Qt5Xml REQUIRED)
 #FIND_PACKAGE(Qt5Help REQUIRED)
+FIND_PACKAGE(Qt5WebKitWidgets REQUIRED)
 FIND_PACKAGE(Qt5PrintSupport REQUIRED)
 
 # in the following lines all the requires include directories are added
diff --git a/src/widgets/helpbrowser.h b/src/widgets/helpbrowser.h
index 57a29d5..0feb406 100644
--- a/src/widgets/helpbrowser.h
+++ b/src/widgets/helpbrowser.h
@@ -12,7 +12,7 @@
 #if QT_VERSION >= 0x050600
 #include <QWebEngineView>
 #else
-#include <QWebView>
+#include <QtWebKitWidgets/QWebView>
 #endif
 
 class CAHelpBrowser
